back/src/composables/useUtils.ts

```diff
@@ -130,6 +147,7 @@ const useUtils = () => ({
   sendResponse,
   toDbFormat,
   fromDbFormat,
+  calculate_distance,
```

Comment:
snake_case whereas other functions are written in camelCase ?

Reply:
It's modified !